input/templates/my-account-edit-address.hbs

```diff
@@ -22,19 +22,19 @@
               </div>
               <div id="my-account-desktop-content" class="col-sm-9">
                 <div class="address-book-content edit-address-book-content">
-                  <form id="form-edit-my-address-book" action="{{@root.meta._links.editMyAddressBook.href}}" method="POST">
+                  <form id="form-edit-my-address-book" action="{{@root.meta._links.myAddressBookEditAddressSubmit.href}}" method="POST">
                     <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
-                    <input type="hidden" name="id" value="{{content.address.id}}"/>
+                    <input type="hidden" name="id" value="{{content.addressForm.id}}"/>
                     <div class="row">
                       <div class="col-sm-12">
                         <div class="address-book-title hidden-xs">
-                          {{i18n "my-account:addressBook.title"}} > {{i18n "my-account:addressBook.normalAddress.title"}} > {{i18n "my-account:buttons.edit"}}
+                          {{i18n "my-account:addressBook.title"}}
```

Comment:
Following the design that we had until now, we never have this structure for subsections. I guess the designers wanted something like this: http://commercetools.github.io/commercetools-sunrise-theme/site/en/my-account-my-orders-order.html (there's the section "My orders" and below a message "Your order details", which would be the subsection). Maybe we can do something like this?